### ISSUE TITLE
Updating adds erroneous line for $PROFILE

### ISSUE THREAD

**Issue by NerdyGriffin** [JanDeDobbeleer/oh-my-posh#673]
### Prerequisites

- [x] I have read and understand the `CONTRIBUTING` guide
- [x] I looked for duplicate issues before submitting this one

### Description

Updating oh-my-posh causes an incorrect line to be added to PowerShell's `$PROFILE` file which causes errors when a new instance of PowerShell is started

### Environment

- Oh my Posh version: 3.140.0
- Theme: sorin
- Operating System: Windows 10
- Shell: PowerShell & PowerShell Core
- Terminal: PowerShell and/or WindowsTerminal (both)

### Steps to Reproduce

1. Update oh-my-posh via chocolatey `choco upgrade oh-my-posh -y --force`
2. Launch a new PowerShell instance. There is an error that says the following:
```
Invoke-Expression : A positional parameter cannot be found that accepts argument '
'.
At D:\Users\Acer\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1:17 char:1
+ Invoke-Expression (oh-my-posh --init --shell pwsh)`n
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Invoke-Expression], ParameterBindingException
    + FullyQualifiedErrorId : PositionalParameterNotFound,Microsoft.PowerShell.Commands.InvokeExpressionCommand
```
3. If you view the powershell profile using `cat $PROFILE`, you will see that the line `` Invoke-Expression (oh-my-posh --init --shell pwsh)`n `` has been appended to `$PROFILE`
4. I then guessed that the `` `n `` is simply a mistaken newline, so I tried removing the `` `n `` so that the line is just `Invoke-Expression (oh-my-posh --init --shell pwsh)`. This creates a new error when you open a new PowerShell instance: 
```
Test-Path : Cannot bind argument to parameter 'Path' because it is an empty string.
At line:13 char:15
+ if (Test-Path $config) {
+               ~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Test-Path], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorEmptyStringNotAllowed,Microsoft.PowerShell.Commands.Test
   PathCommand
```

**Expected behavior:** [What you expected to happen]
Rather obvious in this case: I expect there to be no errors coming from `$PROFILE` when I launch a new PowerShell instance

**Actual behavior:** [What actually happened]

After updating oh-my-posh, launching a new PowerShell instance results in the error I described above:
```
Invoke-Expression : A positional parameter cannot be found that accepts argument '
'.
At D:\Users\Acer\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1:17 char:1
+ Invoke-Expression (oh-my-posh --init --shell pwsh)`n
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Invoke-Expression], ParameterBindingException
    + FullyQualifiedErrorId : PositionalParameterNotFound,Microsoft.PowerShell.Commands.InvokeExpressionCommand
```

## Note:
Some may notice that I mentioned that I installed via chocolatey, and will immediately say that this should be reported to the maintainer of the chocolatey package rather than this repo. Because of this, I specifically looked into this issue in the chocolatey package before bring this bug report here, and from my review of the code so far there is no indication that this bug could be caused by that chocolatey installer rather than being caused by the source (this repo) since the chocolatey package seems to do nothing other than to pull the necessary content from this source and run the `install.exe`.
I will continue to look into this further, and I will comment back here if I find the cause of the problem.

**Comment by NerdyGriffin** [comment]
Okay so the line in question appears to be added by ~line 229 in `src/main.go`, 
```
. . .

switch shell {
    case pwsh:
    return fmt.Sprintf("Invoke-Expression (@(&\"%s\" --print-init --shell=pwsh --config=\"%s\") -join \"`n\")", executable, configFile)
    . . .
}
. . .
```

and the error that occurs after removing the `` `n `` from `$PROFILE` appears to be a line in the file `src/init/omp.ps1` which goes like this
```
. . .

$config = "::CONFIG::"
if (Test-Path $config) {
    $global:PoshSettings.Theme = (Resolve-Path -Path $config).Path
}
. . .
```

I am not familiar with the Go language, so I cannot speak into that code from `src/main.go`, but I am somewhat familiar with PowerShell and I am wondering what is up with the line `$config = "::CONFIG::"`. It is obvious that this line intends to get the path to a config file of some kind, but I do not understand what the `:: . . . ::` syntax is supposed to be doing and it is apparently somehow resolving to nothing (empty string) on my system.
I am curious about this, and I hope this additional information is helpful

**Comment by JanDeDobbeleer** [comment]
@NerdyGriffin `Invoke-Expression (oh-my-posh --init --shell pwsh)` misses the `--config` flag. The ::..:: syntax is just a pointer to replace the actual config provided via that flag. Without it, it resolves to nothing and the script is faulty. You want to load [this](https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/themes/sorin.omp.json) file as the config. When added to your `$HOME`, the syntax looks like this: `Invoke-Expression (oh-my-posh --init --shell pwsh --config ~/sorin.omp.json)`

**Comment by NerdyGriffin** [comment]
Thank you for you feedback, I actually noticed that after I eventually re-found the install instructions (the button/link in the readme displayed super tiny on my screen for some reason, so I did not think to look in those buttons for the documentation, my bad)

I see from the documentation that this use of `Invoke-Expression...` is equivalent to (or an alternative to) using the `Set-PoshPrompt -Theme <theme-name>` syntax, which I already setup in that file months ago. So now I having working and know why it was throwing errors, so thanks again

So I guess the only remaining question is why does an incomplete version of the `Invoke-Expression (oh-my-posh ...)` syntax get appended during the update? It seems like a helpful feature to automatically append that to get people started, though it's weird that it would append an incomplete form rather than maybe setting a default config. (And unnecessary since I already manually set it up in my $PROFILE before the update)
If it turns out that your installer is not setup to append that during the install, then that would means it is a broken features of the choco package, so not a problem in your code.
I think that is the case, because I decided to uninstall the chocolatey package and reinstall using `Install-Module ...` and it did not create this problem
